typescript-test-samples/step-functions-local/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+  "title": "Step Functions Local Testing",
+  "description": "This project shows a technique for testing an AWS Step Functions workflow in a local desktop environment",
+  "content_language": "English",
+  "language": "TypeScript",
+  "type": ["Unit"],
+  "diagram": "/images/stepfunctions_local_test.png",
+  "framework": "SAM",
```

Comment:
Doesn't look like we use SAM here, or am I missing something?

Reply:
What options are available? There is no resources file definition in this project as nothing gets deployed to the cloud.

Reply:
Dropped it